### DISCUSSION TITLE
Error building gulp-protractor:3.0.0 after release @types/[jasmine]

### DISCUSSION THREAD

**Discussion by leonardodamelio** [DefinitelyTyped/DefinitelyTyped#66920] [Issues with a @types package]
Hi Guys, 
I have an old project running on Node 6.4.0 and I'm facing an error building gulp-protractor:3.0.0 after release @types/jasmine: 2.8.20.

This is the error I´m facing:

```
npm ERR! Linux 4.4.0-210-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3
npm ERR! path /project/workspace/node_modules/.staging/@types/jasmine-ef19d1cc/package.json
npm ERR! code ENOTDIR
npm ERR! errno -20
npm ERR! syscall open
```

And this is the package.json file:
```
{
  "name": "test",
  "version": "1.0.0",
  "description": "Testing gulp-protractor",
  "engines": {
    "node": ">=4.3",
    "npm": "~3.0.0"
  },
  "dependencies": {
    "gulp-protractor": "3.0.0"
  }
}
```

[a comment by an automated account is not shown]

**Comment by chivesrs** [comment]
Node 6.4.0 is well out of support so I don't think you're going to get much traction there.

That being said, a quick search on your error code gave https://stackoverflow.com/questions/13452791/npm-install-error-enotdir so try doing that?

**Reply by leonardodamelio** [reply]
The point is that after the release 2.8.20 (published 19 days ago) it stopped working. I tried solution you found on stackoverflow, but without success.
